logbus/setup/setup.go

```diff
@@ -56,17 +59,47 @@ func New(ctx context.Context, bus *logbus.Bus, debug, verbose, forceColor, noCol
 	return bs, nil
 }
 
+// SetOrgAndProject sets the org and project for the manifest.
+func (bs *BusSetup) SetOrgAndProject(orgName, projectName string) {
+	bs.InitialManifest.OrgName = orgName
+	bs.InitialManifest.ProjectName = projectName
+}
+
 // StartLogStreamer starts a LogStreamer for the given build. The
 // LogStreamer streams logs to the cloud.
-func (bs *BusSetup) StartLogStreamer(ctx context.Context, c cloud.Client, orgName, projectName string) {
-	initialManifest := &logstream.RunManifest{
-		BuildId:            bs.BuildID,
-		Version:            deltautil.Version,
-		OrgName:            orgName,
-		ProjectName:        projectName,
-		CreatedAtUnixNanos: uint64(bs.CreatedAt.UnixNano()),
+func (bs *BusSetup) StartLogStreamer(ctx context.Context, c cloud.Client) {
+	bs.LogStreamer = logstreamer.New(ctx, bs.Bus, c, bs.InitialManifest)
+}
+
+// DumpManifestToFile dumps the manifest to the given file.
+func (bs *BusSetup) DumpManifestToFile(path string) error {
+	m := bs.Formatter.Manifest()
+	proto.Merge(m, bs.InitialManifest)
+	f, err := os.OpenFile(path, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0644)
+	if err != nil {
+		return errors.Wrapf(err, "failed to open bus manifest debug file %s", path)
+	}
+	useJson := strings.HasSuffix(path, ".json")
+	var dt []byte
+	if useJson {
+		jsonOpts := protojson.MarshalOptions{
+			Multiline:       true,
+			Indent:          "  ",
+			UseProtoNames:   true,
```

Comment:
nit - feel free to ignore.
Because we are using `lowerCamelCase` for the log storage it would make it a little easier to use the debug file if it was also in `lowerCamelCase`. If there's not good reason for opting for the proto names instead could we swap this?

```suggestion
			UseProtoNames:   false,
```